HTML_CSS/README.md

```diff
@@ -0,0 +1,61 @@
+## HTML/CSS challenge assignments for Wealth42
+
+### Introduction
+The goal of this challenge is to test your familiarity with concepts in HTML & CSS.
+To read more about the recruitment process and open opportunities at Wealth42, click [here](http://bit.ly/w42-hiring)
+
+### Pick an evaluation mode
+
+1.   **Project of your choice**: You are free to submit a project of your own with *some* conditions.
+     *   Qualifying criteria for submission:
+         *   MUST exhibit everything covered in the [Assessment Criteria](#assessment-criteria) and [Instructions](#instructions).
+         *   You are authorised to publish the code as a public pull request via GitHub, following the steps as covered in [Submission Process](#submission-process)
+     *   Additional criteria for submission:
+         *   Attach the story of your project in a file named stories.md at root location. For reference, do checkout [stories](../stories/)
+                >   *Since the project is something we'll need to understand purpose of before evaluating, this file should clearly explain the purpose of the application you are publishing.*
+
+2.   **Wealth42's drafted assignment**: 
+     *   You can select any one of the stories mentioned in the [stories](../stories/) folder, and develop a HTML Website using CSS for it. 
+     *   Follow [Instructions](#instructions) and [Submission Process](#submission-process) to complete the assignment.
+     *   For success, ensure you get as many checks as in our [Assessment Criteria](#assessment-criteria)
+
+### Instructions
+
+*   HTML & CSS: 
+    *   Create upto the mark website based on the user story selected.
+    *   Design and Organise your webpage to depict the frontend and objectives of the story selected.
+    *   Integrate CSS to add styling to the webpage.(Frameworks can be used)
```

Comment:
(Using frameworks is encouraged, however, you must also display excellence in pure CSS as well)